docs/tools/firmware.sh

```diff
@@ -388,7 +388,7 @@ install_package() {
 	local package=$1
 	local package_manager
 	package_manager=$(detect_package_manager)
-	read -rp "Press enter to install $package and its dependencies. Alternatively you can terminate this script by pressing Control+C and install $package yourself, if you want to install it via some alternate method."
+	read -rp "$package is missing. Press enter to install $package and its dependencies. Alternatively you can terminate this script by pressing Control+C and install $package yourself, if you want to install it via some alternate method."
```

Comment:
I actually liked the old wording.

```
Dpkg and/or its dependencies are missing!

Press enter...
```

Reply:
The package's dependencies aren't actually missing so I didnt want to confuse users. If they see that they might think that they have dpkg installed but are missing its dependencies.

Reply:
We tell the user that we are going to install its dependencies in the next sentence. That sentence reads to me like its possible that you might have dpkg but not one of its dependencies, which is not what the script is actually checking

Reply:
> We tell the user that we are going to install its dependencies in the next sentence. That sentence reads to me like its possible that you might have dpkg but not one of its dependencies, which is not what the script is actually checking

```
Dpkg is missing!

Press enter...
```

Then do this